src/doc/unstable-book/src/library-features/asm.md

```diff
@@ -372,6 +372,43 @@ unsafe {
 # }
 ```
 
+## Labels
+
+The compiler is allowed to instantiate multiple copies an `asm!` block, for example when the function containing it is inlined in multiple places. As a consequence, you should only use GNU assembler [local labels] inside inline assembly code. Defining symbols in assembly code may lead to assembler and/or linker errors due to duplicate symbol definitions.
+
+Moreover due to [a llvm bug], you cannot use `0` or `1` as labels. Therefore only labels in the `2`-`99` range are allowed.
```

Comment:
It's not just 0 and 1. It's actually anything that can be parsed as a binary literal, so any number that contains only 0 and 1 digits.

Reply:
Indeed. I made it clear.